zhmc_prometheus_exporter/zhmc_prometheus_exporter.py

```diff
@@ -2424,6 +2457,11 @@ def main():
                 hmc_features_str = ', '.join(hmc_features) or 'None'
                 logprint(logging.INFO, PRINT_V,
                          f"HMC features: {hmc_features_str}")
+
+                cpc_names = [cpc.name for cpc in cpc_list]
```

Comment:
Is this recomputation of cpc_names necessary? I don't see that the cpc_list would have changed.

Reply:
Not necessary and removed. Done.